### PR TITLE
Change var and let where necessary

### PR DESCRIPTION
**Pull request by schoenwaldnils**
As discussed in #31 here my implementation of `const` and `let`.

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
Unfortunately, now we have at least two concurrent full attempts (https://github.com/ryanmcdermott/clean-code-javascript/pull/38 and https://github.com/ryanmcdermott/clean-code-javascript/pull/51) and two partial attempts (https://github.com/ryanmcdermott/clean-code-javascript/pull/33 and https://github.com/ryanmcdermott/clean-code-javascript/pull/44) to address the issue.

**Comment by schoenwaldnils** [comment]
#33 is more an attempt to show the use of the variable-name, isn't it?
#44 is included in #51 

#38 defaults to `let`. 
In my understanding `let` only should be used where the variable is reassigned.
But `const`can be alternated.
For example the eslint-decleration [prefer-const](http://eslint.org/docs/rules/prefer-const)

File: README.md [hunk 1]
```diff
@@ -1693,19 +1692,19 @@ const assert = require('assert');
...
   it('handles 30-day months', function() {
-    let date = new MakeMomentJSGreatAgain('1/1/2015');
+    const date = new MakeMomentJSGreatAgain('1/1/2015');
     date.addDays(30);
     date.shouldEqual('1/31/2015');
   });
 
   it('handles leap year', function() {
-    let date = new MakeMomentJSGreatAgain('2/1/2016');
+    cosnt date = new MakeMomentJSGreatAgain('2/1/2016');
```
Comment (ryanmcdermott) [inline README.md]: Should be `const`

File: README.md [hunk 2]
```diff
@@ -1693,19 +1692,19 @@ const assert = require('assert');
...
     date.addDays(28);
     assert.equal('02/29/2016', date);
   });
 
   it('handles non-leap year', function() {
-    let date = new MakeMomentJSGreatAgain('2/1/2015');
+    cosnt date = new MakeMomentJSGreatAgain('2/1/2015');
```
Comment (ryanmcdermott) [inline README.md]: Should be `const`

File: README.md (hunk 1 above)
Comment (schoenwaldnils) [inline README.md]: my bad :D

File: README.md (hunk 2 above)
Comment (schoenwaldnils) [inline README.md]: Is fixed now

**Comment by ryanmcdermott** [comment]
This looks great! Since sadly the examples in this document aren't testable (maybe we could find a way in the future), I assume that everything here works after reading it as carefully as I could. If anyone sees anything else that stands out with these changes, open an issue or ideally a PR 😄

**Pull request merged**